src/integrationTest/java/uk/gov/hmcts/darts/audio/service/ViqHeaderServiceTest.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.darts.audio.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.darts.audio.service.impl.ViqHeaderServiceImpl;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+
+
+@SuppressWarnings({"PMD.TestClassWithoutTestCases"})
+class ViqHeaderServiceTest extends IntegrationBase {
+
+    @Autowired
+    ViqHeaderServiceImpl viqHeaderService;
```

Comment:
FYI: For an `integrationTest` you would normally just autowire the interface - Spring will know about the implementing class

Reply:
It was a cut and paste error as I copied the unit test to integration test